smsclient_ovh/models/keychain.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+# Copyright (C) 2015 Sébastien BEAU <sebastien.beau@akretion.com>
+# Valentin CHEMIERE <valentin.chemiere@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+OVH_KEYCHAIN_NAMESPACE = 'ovh_provider'
+
+
+class AccountProduct(models.Model):
```

Comment:
[Please respect file and model naming conventions](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming)